Sources/Public/Animation/LottieAnimationViewInitializers.swift

```diff
@@ -78,6 +78,31 @@ extension LottieAnimationView {
     }
   }
 
+  /// Loads a Lottie animation asynchronously from the URL
+  ///
+  /// - Parameter url: The url to load the animation from.
+  /// - Parameter imageProvider: An image provider for the animation's image data.
+  /// If none is supplied Lottie will search in the main bundle for images.
+  @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
+  public convenience init(
+    url: URL,
+    imageProvider: AnimationImageProvider? = nil,
+    animationCache: AnimationCacheProvider? = LottieAnimationCache.shared,
+    configuration: LottieConfiguration = .shared) async throws
+  {
+    if let animationCache = animationCache, let animation = animationCache.animation(forKey: url.absoluteString) {
+      self.init(animation: animation, imageProvider: imageProvider, configuration: configuration)
+    } else {
+      self.init(animation: nil, imageProvider: imageProvider, configuration: configuration)
+
+      guard let animation = await LottieAnimation.loadedFrom(url: url, animationCache: animationCache) else {
+        throw LottieDownloadError.downloadFailed
+      }
```

Comment:
It seems like we should propagate the error from the `loadedFrom` call, as-is it seems to be discarded outright which makes it impossible to get any insight into this error